patroni/ha.py

```diff
@@ -689,7 +702,9 @@ def _is_healthiest_node(self, members, check_replication_lag=True):
     def is_failover_possible(self, members, check_synchronous=True, cluster_lsn=None):
         ret = False
         cluster_timeline = self.cluster.timeline
-        members = [m for m in members if m.name != self.state_handler.name and not m.nofailover and m.api_url]
+        is_static_primary = self.is_static_primary()
+        members = [m for m in members if m.name != self.state_handler.name
+                   and not m.nofailover and m.api_url and not is_static_primary]
```

Comment:
Another nit, what if we bail out quickly on `is_static_primary` with a decent log message?

```suggestion
        if self.is_static_primary():
            logger.warning('manual failover: members list is empty')
            return ret
```
And leave the rest of the code intact?

Reply:
That's a good idea. Will do.

Reply:
Done.